main/src/main/java/org/qbicc/main/Main.java

```diff
@@ -414,10 +414,11 @@ void call0(BaseDiagnosticContext initialContext) {
                                 builder.addResolverFactory(InternalNativeTypeResolver::new);
                                 builder.addResolverFactory(NativeTypeResolver::new);
 
-                                builder.addTaskWrapperFactory(Phase.ADD, next -> (wrapper, ctxt) -> {
+                                // from this point on, all pre-ADD hook tasks are run within a VM thread
```

Comment:
Perhaps; they do set special names so that might be of sufficient value that it's worth keeping them, but at the same time, simpler is better - I could go either way on it.

Reply:
ok.  just wanted to be sure I wasn't missing something.